Update compression docs to reflect 2.0 changes (#515)
timescale/docs.timescale.com-content: using-timescaledb/compression.md

``` sql
SELECT *
FROM chunk_compression_stats('conditions');
```

| chunk_schema | chunk_name | compression_status | before_compression_table_bytes | before_compression_index_bytes | before_compression_toast_bytes | before_compression_total_bytes | after_compression_table_bytes | after_compression_index_bytes | after_compression_toast_bytes | after_compression_total_bytes | node_name |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| _timescaledb_internal | _hyper_1_1_chunk | Compressed | 8192 bytes | 16 kB | 8192 bytes |  32 kB  | 8192 bytes   | 16 kB  | 8192 bytes | 32 kB | |

> | _timescaledb_internal | _hyper_1_20_chunk | Uncompressed | | | | | | | | | |

Reviewer: Given the width, perhaps these should be shown in `\x` mode?

Author: I think in this instance, printing in `\x` would be highlighting the wrong thing.  Really we're just demonstrating the command, the actual numbers in the result aren't interesting, so I feel like minimizing the vertical space and allowing the user to scroll is a better fit.

To accomplish this we first find the job_id of the policy using:

```sql

> SELECT s.job_id
> FROM timescaledb_information.jobs j
> INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
> WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;

Reviewer: ```suggestion
SELECT s.job_id
  FROM timescaledb_information.jobs j
  INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
  WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```

Reviewer 2: @mfreed Your proposal is not consistent with other examples in this and other docs. The top select clauses are usually indented in the same way as `SELECT`, however the join should have an additional indentation. I believe it should be:
```suggestion
SELECT s.job_id
FROM timescaledb_information.jobs j
  INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```
or
```suggestion
SELECT s.job_id
FROM timescaledb_information.jobs j
     INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
WHERE j.proc_name = 'policy_compression' AND s.hypertable_name = <target table>;
```

Author: Doing a quick scan of the docs, it looks like there isn't a standard usage here.  Most common appears to be having nothing indented, though there are some examples where only joins are indented and a few others where everything is indented.  Personally I find Ruslan's suggestion a bit more readable than Mike's here (though I find the larger indentation a bit glaring), so I'm going with that for the moment.  I expect we'll have to revisit this if we decide to standardize our docs sql.

Author: Looking at that example doc, it does look like there's a effort to make them more readable.  However, there still doesn't seem to be much consistency between examples.

our compression policy job:

``` sql
SELECT alter_job(<job_id>, scheduled => true);
```

This job will re-compress any chunks that were decompressed during your backfilling

> operation the next time it runs. To have it run immediately you can do the following:

Reviewer: ```suggestion
operation the next time it runs. To have it run immediately, you can expressly execute
the command via [`run_job`](/api#run_job):
```

our compression policy job:

``` sql
SELECT alter_job(<job_id>, scheduled => true);
```

This job will re-compress any chunks that were decompressed during your backfilling

> operation the next time it runs. To have it run immediatelyimmediately, you can expressly execute

Reviewer: ```suggestion
operation the next time it runs. To have it run immediately, you can expressly execute
```

Author: fixed
